runtime/compiler/control/RecompilationInfo.hpp

```diff
@@ -441,6 +452,11 @@ class TR_PersistentJittedBodyInfo
 
    // ### IMPORTANT ###
 
+
+#if defined(J9VM_OPT_MICROJIT)
+   bool _isMJITCompiled;
```

Comment:
Instead of `bool _isMJITCompiled` you could use a bit in the `_flags`. Bit 0x2 is available